examples/howto/marginalizing-models.ipynb

```diff
@@ -0,0 +1,1440 @@
+{
```

Comment:
<p>But if you insist on using the NUTS sampler *exclusively*.</p><p> </p><p>Link to wiki on Rao-Blackwell's theorem, and maybe only bring it up after explaining the advantages of marginalizing variables: more efficient sampling, unclear statistical properties of mixed sampling...</p><p> </p><p>get a lower variance estimate of your parameters is a bit fuzzy. Get a smaller monte-carlo error on the estimates?</p><p> </p><p>Call it <code>pymc_experimental</code> not <code>pmx</code> and maybe mention already the <code>MarginalModel</code> class with a cross-reference to the respective docs page   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/628/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
What's the correct way to cross-reference the `pymc_experimental` docs?

Reply:
You can see how cross-references work in https://github.com/pymc-devs/pymc-examples/pull/614.

Reply:
Also https://github.com/pymc-devs/pymc-examples/issues/605